Quorum/apis/llms/llm_conversation.py

```diff
@@ -0,0 +1,105 @@
+from enum import StrEnum, auto
+from pathlib import Path
+
+class MessageRole(StrEnum):
+    USER = auto()
+    ASSISTANT = auto()
+
+
+MAX_TOKENS = 1024
+
+
+def tmp_send_message_to_claude(client, model, messages):
+    try:
+        response = client.messages.create(
+            max_tokens=MAX_TOKENS,
+            model=model,
+            messages=messages
+        )
+    except:
+        raise
+
+    with open(Path(__file__).parent / 'ClaudeExecutionsNum.txt', 'r+') as f:
+        num = int(f.read())
+        f.seek(0)
+        f.write(str(num + 1))
+        f.truncate()
```

Comment:
This solution doesn't support future extensions for async usage of the model. which is one of the downsides of not using out-of-the-box infrastructure.

Reply:
This whole function is temporary and I will probably remove it before merging this PR. I made sure to make it as ugly as possible for this reason. This is just because @yoav-el-certora asked me to notice how many API calls I make during development so we estimate costs and I didn't want to write manually every time I call the LLM.

Reply:
This is just to understand the costs while implementing, and its temporary only